test/Microsoft.DotNet.ShellShimMaker.Tests/LinuxEnvironmentPathTests.cs

```diff
@@ -0,0 +1,85 @@
+// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Runtime.InteropServices;
+using FluentAssertions;
+using Microsoft.DotNet.Tools.Test.Utilities;
+using Microsoft.Extensions.DependencyModel.Tests;
+using Xunit;
+
+namespace Microsoft.DotNet.ShellShimMaker.Tests
+{
+    public class LinuxEnvironmentPathTests
+    {
+        [Fact]
+        public void GivenEnvironementAndReporterItCanPrintOutInstructionToAddPath()
```

Comment:
`Environement`is mis-spelled. Should say `Environment`.

This seems to be the case in a bunch of places in this file.

Reply:
all fixed